### PR TITLE
feat(server): Expose serialization bytes via `INFO` and `/metrics`

### PR DESCRIPTION
**Pull request by chakaz**

### CODE REVIEW

File: src/server/server_family.cc [hunk 1]
```diff
@@ -1107,6 +1107,8 @@ void PrintPrometheusMetrics(const Metrics& m, DflyCmd* dfly_cmd, StringResponse*
                            << sdata_res.error().message();
   }
   AppendMetricWithoutLabels("tls_bytes", "", m.tls_bytes, MetricType::GAUGE, &resp->body());
+  AppendMetricWithoutLabels("serialization_bytes", "", m.serialization_bytes, MetricType::GAUGE,
```
Comment (romange) [inline src/server/server_family.cc:1110]: is it full sync serialization_bytes ? stable state as well?

Comment (chakaz) [reply src/server/server_family.cc:1110]: nope, this is only `SliceSnapshot`, i.e. only full sync and *not* stable sync

Comment (romange) [reply src/server/server_family.cc:1110]: ok, let's maybe call it "full_sync_serialization_bytes" here and in `/metrics` ?

Comment (chakaz) [reply src/server/server_family.cc:1110]: The thing is - `SliceSnapshot` is also used by `SAVE`, so `full_sync_*` might be a bit misleading

Comment (chakaz) [reply src/server/server_family.cc]: Done :)

**Pull request merged**